ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -392,6 +392,19 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
                                                    project_id=project.project_id,
                                                    sample_id=sample.name)
     sbatch_text_list.append("\nPIPER_RETURN_CODE=$?")
+
+    #Precalcuate md5sums
+    sbatch_text_list.append('MD5FILES="$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.bam'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.bai'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.metrics'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/07_variant_calls/*.genomic.vcf.gz"'.format(project.project_id))
```

Comment:
I think that in a previous comment @b97pla suggested to compute md5 on all vcf files...